babel/messages/extract.py

```diff
@@ -400,56 +401,65 @@ def extract(
                    options=options or {})
 
     for lineno, funcname, messages, comments in results:
-        spec = keywords[funcname] or (1,) if funcname else (1,)
+        specs = keywords[funcname] or (1,) if funcname else (1,)
```

Comment:
Might as well simplify this to
```suggestion
        specs = keywords.get(funcname) or (1,)
```
(assuming `keywords[x]` is never supposed to be falsy)?

Reply:
Sure thing. :)